data/ecosystem/adopters.yaml

```diff
@@ -14,6 +14,11 @@
   components: [Collector, Go, Java, JavaScript, .NET]
   reference: ''
   contact: ''
+- name: CloudRaft
+  url: https://www.cloudraft.io
+  components: [Collector, Go, Python, JavaScript]
+  reference: 'https://www.cloudraft.io/blog/cicd-observability-using-opentelemetry'
```

Comment:
see my comment around the blog post not being a case study/an adopter story but a "how to"

Reply:
Sure, will re-do and share along with the suitable docs. Thanks.